R/tax_range_time.R

```diff
@@ -157,7 +173,12 @@ tax_range_time <- function(occdf,
     if (by == "LAD") {
       temp_df <- temp_df[order(temp_df$min_ma), ]
       temp_df$taxon_id <- seq_len(nrow(temp_df))
-      }
+    }
+
+    if (is.null(group)) {
+      temp_df <- temp_df[order(temp_df$group), ]
```

Comment:
Currently, we're ordering by `group` after ordering `name` by how the user specified. I think when group is supplied, we should be ordering using `by` for `name` and `group` (presumably `group` first), e.g. `group` are all organised by FAD, and then name within `group` are organised by FAD too.